appcues/src/main/java/com/appcues/data/mapper/experience/ExperienceMapper.kt

```diff
@@ -40,6 +43,10 @@ internal class ExperienceMapper(
             priority = priority,
             type = from.type,
             publishedAt = from.publishedAt,
+            completionActions = arrayListOf<ExperienceAction>().apply {
```

Comment:
this is where I'm mapping from the remote data and creating the ExperienceActions as a list. (the order here matters)